app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/common/latest_projects_list/LatestProjectsList.jsx

```diff
@@ -12,7 +12,7 @@ const LatestProjectsList = () => {
   return (
     <div className="space-y-3">
       <h3 className="font-semibold">{latest_projects}</h3>
-      <div className="mt-12 flex flex-col mx-0 md:flex-row lg:flex-row m-2 justify-center">
+      <div className="mt-12 flex flex-col mx-0 t:flex-row m-2 justify-center">
```

Comment:
Hittade att "md" enligt tailwind.config ej ska användas, utan "t" istället. Tog även bort deklaration för "lg" då "md" gör att alla case från och med storlek "md" anammar beteendet, därför behöver inte större skärmar med samma värde deklareras.

Reply:
Okej najs 👍

Reply:
Okej, var hittade du detta? Påverkar min task!

Reply:
Det står i config-filen med en kommentar att "below are temporary". Det tänkte jag innebar att de inte skulle prioriteras, utan att "t" var det etablerade värdet

Reply:
oki! La in en task för att fixa det för alla filer i planner!